lib/Scheduler/aco.cpp

```diff
@@ -401,21 +401,21 @@ static void PrintInstruction(SchedInstruction *inst) {
   std::cerr << std::setw(20) << std::left << inst->GetOpCode();
 
   std::cerr << " defs ";
-  Register **defs;
-  uint16_t defsCount = inst->GetDefs(defs);
-  for (uint16_t i = 0; i < defsCount; i++) {
-    std::cerr << defs[i]->GetNum() << defs[i]->GetType();
-    if (i != defsCount - 1)
+  int i = 0;
+  for (const Register *def : inst->GetDefs()) {
```

Comment:
Maybe don't use a range based loop here?

Reply:
I moved to `llvm::enumerate(...)`.